lib/l10n/intl_it.arb

```
@@ -611,6 +611,38 @@
+  "@_comment_about_screen_new": "About Screen New Strings",
+  "documentation": "Documentazione",
+  "usersDocumentationEnglish": "Documentazione Utente (Inglese)",
+  "usersDocumentationSpanish": "Documentazione Utente (Spagnolo)",
+  "technicalDocumentation": "Documentazione Tecnica",
+  "read": "Leggi",
+  "technicalDetails": "Dettagli Tecnici",
+  "mostroDaemonVersion": "Versione Daemon Mostro",
+  "mostroCommitId": "ID Commit Mostro",
+  "orderExpiration": "Scadenza Ordine",
+  "holdInvoiceExpiration": "Scadenza Fattura di Blocco",
+  "holdInvoiceCltvDelta": "Delta CLTV Fattura di Blocco",
```

RuboCop (1.76.1)

[warning] Duplicated key in hash literal.

(Lint/DuplicateHashKey)